build/tester/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/helmpack/chart-testing:v3.3.1
+
+COPY --from=golang:1.15-alpine /usr/local/go/ /usr/local/go/
+ENV PATH="/usr/local/go/bin:${PATH}"
+
+# Install gcc.
+RUN apk add build-base
+
```

Comment:
@jlinder Can you take a look at this and let me know what you think? Also is there a way to cache the go dependencies listed in `go.mod` without running `go test`? I'm thinking of something like `npm install`

Reply:
I think what you're looking for is `go mod vendor`. It will create a `vendor` directory which you can check-in to source control.